src/pages/SearchPage.js

```diff
@@ -101,17 +105,37 @@ class SearchPage extends Component {
      * @returns {Array}
      */
     getSections() {
+        return this.filterAdapter('');
+    }
```

Comment:
I think you need to rename this method / update the comment now

Reply:
I renamed it to searchSections.